components/cluster/command/display.go

```diff
@@ -44,11 +48,22 @@ func newDisplayCmd() *cobra.Command {
 			}
 
 			clusterName = args[0]
-			if err := displayClusterMeta(clusterName, &gOpt); err != nil {
-				return err
+
+			if tiuputils.IsNotExist(meta.ClusterPath(clusterName, meta.MetaFileName)) {
+				return errors.Errorf("Cluster %s not found", clusterName)
 			}
-			if err := displayClusterTopology(clusterName, &gOpt); err != nil {
-				return err
+
+			if showDashboardOnly {
+				if err := displayDashboardInfo(clusterName); err != nil {
+					return err
+				}
+			} else {
```

Comment:
I think we can return in `showDashboardOnly` and avoid the `else` block

Reply:
There is a tear down step `destroyTombstoneIfNeed`, is it ok to skip?

Reply:
In this case, I think it's ok to skip. This step is used to destroy tombstone state tikv,  it may take some time. It will be confusing to wait such a time for `--dashboard`

Reply:
Resolved, PTAL again, thanks!